src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerServiceTest.java

```diff
@@ -644,6 +643,32 @@ public void midEventAmendHearingDateOnWeekend() {
         assertEquals(LISTED_DATE_ON_WEEKEND_MESSAGE + hearingNumber, errors.get(0));
     }
 
+    @Test
+    public void midEventAmendHearingDateInPast() {
```

Comment:
These tests are a good example of what Mehmet was talking about yesterday. This could be changed into a ParameterizedTest

`@CsvSource({"Listed, Yes", "Heard, No"})`